CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,3 @@
 
+-  Allow access entities using NGSIv2 API for non_signal rules (#549)
```

Comment:
```suggestion
-  Allow access entities using NGSIv2 API for non_signal rules (new setting nonSignalByAPI / PERSEO_CHECK_NON_SIGNAL_BY_API) (#549)
```

Reply:
solved in this commit: https://github.com/telefonicaid/perseo-fe/pull/726/commits/18c2c89094863182a06bf6fe459ac414531887da